circle.yml

```diff
@@ -5,7 +5,7 @@ deployment:
   production:
     branch: master
     heroku:
-      appname: hylo-node
+      appname: hylo-node-hitfin
```

Comment:
hmm, i'm not going to want to merge this into master. Can you try setting it up as an environment variable or otherwise parameterizing it?